addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChatWindow" owl="1">
-        <div class="o_ChatWindow" tabindex="0"
+        <div class="o_ChatWindow" tabindex="0" t-att-data-visible-index="chatWindow.visibleIndex"
```

Comment:
Should guard in case `chatWindow` is `undefined` (async issue with models and OWL rendering that we are aware of, hence the `chatWindow and chatWindow.[...]` and the global `t-if="chatWindow"` you see in this template).